core/src/main/scala/kafka/log/Log.scala

```diff
@@ -762,7 +766,7 @@ class Log(@volatile var dir: File,
         updateFirstUnstableOffset()
 
         trace("Appended message set to log %s with first offset: %d, next offset: %d, and messages: %s"
```

Comment:
Maybe we should just use the accurate last offset instead of a potentially inaccurate first offset? Or perhaps we could print both of them, but use the `Option` for the first offset?

Also nit: can we change this log message to use string interpolation?

Reply:
Done